Restore backtrace support.

===================================================================
RCS file: /cvs/xorg/xserver/xorg/include/xorg-config.h.in,v
rcsdiff: /cvs/xorg/xserver/xorg/include/xorg-config.h.in,v: warning: Unknown phrases like `commitid ...;' are present.
retrieving revision 1.12
retrieving revision 1.13
diff -u -r1.12 -r1.13
--- xorg-server-1.0.1.orig/xorg/include/xorg-config.h.in	2005/12/02 06:02:44	1.12
+++ xorg-server-1.0.1/xorg/include/xorg-config.h.in	2006/01/26 00:23:44	1.13
@@ -106,4 +106,7 @@
 /* System has wscons console */
 #undef WSCONS_SUPPORT
 
+/* Has backtrace support */
+#undef HAVE_BACKTRACE
+
 #endif /* _XORG_CONFIG_H_ */
